ocsptools/helpers/ocsp_tools_helpers.go

```diff
@@ -0,0 +1,135 @@
+package helpers
+
+//go:generate mockgen -source=ocsp_tools_helpers.go -destination=../../mocks/helpersmock/mock_ocsptoolshelpers.go -package=helpersmock
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"golang.org/x/crypto/ocsp"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+const (
+	RespTimeLimit    = "10s"
+	TimeoutInSeconds = 20
```

Comment:
Is there a reason why one is a string and the other is an int?

Reply:
RespTimeLimit is a string because it needs to be parsed into a `time.Duration` object (as far as I know there is no clean way to declare it as a duration automatically since it's a constant and therefore can't be declared as the return value of a function, if you remember there was a previous comment about this as well)

Reply:
I guess I was just wondering if `TimeoutInSeconds` should be a string `"20s"` and then get parsed to 20 by the same library you are using to parse `RespTimeLimit`. But it isn't necessary.

Reply:
So `TimeoutInSeconds` is used in the http client (shouldn't be parsed into a duration, should just be an int) whereas the `RespTimeLimit` is parsed into a duration to be used by the time package.